internal/core/metadata/application/notify.go

```diff
@@ -66,6 +72,8 @@ func updateDeviceCallback(ctx context.Context, dic *di.Container, serviceName st
 	if response.StatusCode != http.StatusOK {
 		lc.Errorf("fail to invoke device service callback for updating device %s, err: %s", device.Name, response.Message)
 	}
+
+	sendNotification(ctx, dic, device.Name, http.MethodPatch)
```

Comment:
Use `Device update` instead of `http.MethodPatch`

Reply:
Modified.